skopeo.spec.rpkg

```diff
@@ -9,8 +9,11 @@
 # CAUTION: This is not a replacement for RPMs provided by your distro.
 # Only intended to build and test the latest unreleased changes.
 
-%global gomodulesmode GO111MODULE=on
-%global with_debug 1
+# TODO: enable GO111MODULE=off
+%define gomodulesmode GO111MODULE=on
+%define gobuild(o:) %{gomodulesmode} go build -buildmode pie -compiler gc -tags="rpm_crashtraceback ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '-Wl,-z,relro -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '" -a -v -x %{?**};
+# TODO: enable debuginfo
+%global with_debug 0
```

Comment:
It’s unclear to me how `with_debug` relates. Is because this `gobuild` is rather different from the `go-rpm-macros` one, and that breaks builds with debug info?